dora/core/client/fs/src/test/java/alluxio/client/file/ufs/UfsBaseFileSystemTest.java

```diff
@@ -357,17 +346,17 @@ public void writeThenGetStatus() throws IOException, AlluxioException {
     Assert.assertEquals(chunkSize, mFileSystem.getStatus(uri).getLength());
   }
 
-/*  @Test
+  @Test
   public void getStatusWhenWriting() throws IOException, AlluxioException {
     AlluxioURI uri = mRootUfs.join("getStatusWhenWriting");
     int chunkSize = 512;
     try (FileOutStream outStream = mFileSystem.createFile(uri)) {
+      // TODO(Yichuan): Please verify its correctness.
       outStream.write(BufferUtils.getIncreasingByteArray(chunkSize));
-      // local & S3 both failed here
-      Assert.assertEquals(chunkSize, mFileSystem.getStatus(uri).getLength());
       outStream.write(BufferUtils.getIncreasingByteArray(chunkSize, chunkSize));
+      outStream.close();
```

Comment:
if ourstream close here, can we remove the try with resources?
because try with FileOutStream suppose to close the outstream at the end

Reply:
why move close up?

Reply:
If a FileOutStream is writing to a file, the getStatus() cannot get the file status until FileOutStream closed, so I add a stream.close() here. If you getStatus() by force before the stream closed, the return value is not correct.
My question is, the test method is called "getStatusWhenWriting", meaning we call getStatus() when a stream is writing, while now it is impossible as I have said. So how about remove this method? Or rename it?

Reply:
Yes, it is strange to close the resource early in a try-catch, I will modify it later.